ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -256,6 +257,7 @@ descriptor:
   user: "A person who uses or operates something."
   vapor: "A substance in the gas phase at a temperature lower than its critical temperature."
   ventilation: "Process used to provide fresh (typically atmospheric) air into a system or zone."
+  waste: "Substance generated after the use in a variety of deliberate applications or processes."
```

Comment:
reword to "An undesirable substance gathered in a fluid as it goes through deliberate applications or processes"